dev/com.ibm.websphere.appserver.features/visibility/public/jpaContainer-3.0/com.ibm.websphere.appserver.jpaContainer-3.0.feature

```diff
@@ -0,0 +1,26 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.jpaContainer-3.0
+visibility=public
+singleton=true
+kind=beta
+edition=core
+Subsystem-Name: Jakarta Persistence API Container 3.0
+IBM-ShortName: jpaContainer-3.0
+IBM-API-Package: jakarta.persistence; type="spec", \
+ jakarta.persistence.spi; type="spec", \
+ jakarta.persistence.criteria; type="spec", \
+ jakarta.persistence.metamodel; type="spec"
+IBM-App-ForceRestart: uninstall, \
+ install
+-features=com.ibm.websphere.appserver.jndi-1.0, \
+ com.ibm.websphere.appserver.classloading-1.0, \
+ com.ibm.websphere.appserver.optional.jaxb-2.2; ibm.tolerates:=2.3, \
```

Comment:
eventually this will need to be JAX-B 3.0, and we should make it a regular dependency instead of an optional one

Reply:
Will do once it's available.